src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -1328,9 +1328,13 @@ bool ResumptionDataDB::SelectImageData(
   }
   /* Position of data in "select_image" :
      field "imageType" from table "image" = 0
-     field "value" from table "image" = 1*/
+     field "value" from table "image" = 1
+     optional field "isTemplate" from table "image" = 2 */
   image[strings::image_type] = select_image.GetInteger(0);
   image[strings::value] = select_image.GetString(1);
+  if (!select_image.IsNull(2)) {
```

Comment:
isTemplate is optional field.
@LuxoftAKutsan, I rewrite last commit
For now it check if field exist.